lib/rules/indent.js

```diff
@@ -888,11 +880,11 @@ module.exports = {
                 // alter the expectation of correct indentation. Skip them.
                 // TODO: Add appropriate configuration options for variable
                 // declarations and assignments.
-                if (getVariableDeclaratorNode(node)) {
+                if (getParentNodeByType(node, "VariableDeclarator", "FunctionExpression")) {
```

Comment:
Sorry @platinumazure @not-an-aardvark used Array.includes in my solution but forgot it wasnt supported, reverted my changes and re pushed.  Will have a solution up tonight.  Just wanted to give you a heads up.

Thanks